clrfund-constitution.md

```diff
@@ -1,11 +1,11 @@
 # CLR.FUND CONSTITUTION
 
 ## I. Introduction
-Clr.fund is a protocol for efficiently allocating funds to public goods that benefit the Ethereum Network according to the preferences of the Ethereum Community. Clr.fund strives for credible neutrality, decentralization, permissionlessness, trustlessness, and pseudonymity.
+Clr.fund is a protocol for efficiently allocating funds to public goods. Clr.fund strives for credible neutrality, decentralization, permissionlessness, trustlessness, and pseudonymity.
 
 This document outlines clr.fund’s purpose, the properties it must have in order to fully achieve that purpose, and the principles guiding its development.
 
-*Note: clr.fund is open-source software and may be forked by any community looking to allocate funds to public goods. This document pertains to the instance of clr.fund dedicated to public goods benefiting the Ethereum Ecosystem. Other instances may have different objectives or values, and this document does not represent them.*
+This document describes the instance of clr.fund which goal is allocation of funds to public goods that benefit the Ethereum Network according to the preferences of the Ethereum Community, but anyone can deploy their own instance and use it for fund allocation in all kinds of communities and ecosystems.
```

Comment:
I think "which" should be replaced with "whose"
> This document describes the instance of clr.fund ~which~**whose** goal

Or we should restructure the sentence like this
> The goal of the instance of clr.fund described in this document is the allocation...

Reply:
>The goal of the instance of clr.fund described in this document is the allocation...

I changed to this variant